wishlist-backend/wishlist/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from wishlist.models import Wishlist
+# Register your models here.
```

Comment:
can delete the line.

Reply:
These lines help us to perform CRUD operations in built-in django admin panel. (Remember I showed you admin panel in our last meeting)